src/containers/NamespacesDropdown/NamespacesDropdown.stories.js

```diff
@@ -44,7 +44,11 @@ storiesOf('NamespacesDropdown', module)
     });
     return (
       <Provider store={store}>
-        <NamespacesDropdown {...props} />
+        <NamespacesDropdown
+          {...props}
+          titleText="Namespace"
+          label="Namespace"
```

Comment:
We already have `titleText` so let's remove the label here since we'll really only use one or the other.
It would also be good to set `selectedItem={{ text: 'default' }}` to showcase that functionality here.

Reply:
given the reducer is responsible for setting the default value and the `NamespacesDropdown` component interacts with the reducer I'm going to leave out `selectedItem={{ text: 'default' }}`